src/content/docs/dropins/checkout/containers/payment-methods.mdx

```diff
@@ -9,6 +9,8 @@ The `PaymentMethods` container is designed to manage and display the available p
 
 ## PaymentMethods configurations
 
+🚧 TODO: Update to take into account changes to API made in https://git.corp.adobe.com/Commerce-Storefront/storefront-checkout/pull/963 (with new `slots: {Methods: []}` structure) 🚧
```

Comment:
we should remove this entry, no?. I don't expect to see a TODO with a PR link in our docs

Reply:
We'll have to rewrite that section before merging into boilerplate. The TODO is for us not to forget.

Note that the target branch of this pull request is your `payments-develop` branch. I propose merging this pull request, and tackle this with the help of the Thunderbolts team before merging https://github.com/commerce-docs/microsite-commerce-storefront/pull/170.

Reply:
no problem, as mentioned in slack, better remove this for the moment in our docs, I'll put a note into the `payments-develop` branch